pkg/cluster/database.go

```diff
@@ -243,3 +273,78 @@ func makeUserFlags(rolsuper, rolinherit, rolcreaterole, rolcreatedb, rolcanlogin
 
 	return result
 }
+
+// Creates a connection pool credentials lookup function in every database to
+// perform remote authentification.
+func (c *Cluster) installLookupFunction(poolSchema, poolUser string) error {
+	var stmtBytes bytes.Buffer
+	c.logger.Info("Installing lookup function")
+
+	if err := c.initDbConn(); err != nil {
+		return fmt.Errorf("could not init database connection")
+	}
+	defer func() {
+		// in case if everything went fine this can generate a warning about
+		// trying to close an empty connection.
```

Comment:
seen this, too. Can't you simply remove the `closeDBConn` block at the end of this function?